themes/theme-ios11/locale/nl-NL.json

```diff
@@ -132,7 +132,6 @@
     "pick_an_attribute": "Kies een {0}",
     "tax_disclaimer": "* Inbegrepen BTW plus verzendkosten indien van toepassing",
     "item_added": "{count, plural, =0 {# no items added} =1 {{count} item added} other {{count} items added}}",
-    "adding_item": "{count, plural, =1 {{count} artikel toevoegen} other {{count} artikelen toevoegen}}",
```

Comment:
so you deleted those three lines with the variables? we want to have the corrected version I think..

Reply:
fixed